app/web/markdown/volunteer/ui-ux-designer.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "UI/UX Designer"
+description: "Apply to become a UI/UX Designer for Couchers.org"
+---
+
+## [Apply Here](/volunteer/form)
+
+## Position Description
+
+**This is a remote volunteer position**
+
+Couchers.org is seeking an experienced UI/UX Designer to help us build and improve upon next-generation, non-profit couch surfing platform. We need help refining and iterating on our existing Figma-based designs for web and mobile platforms. We’re looking for a candidate with substantial experience in ui/ux design of web and/or mobile apps with a modern, minimalist design. Our designs are already well-established, so much of the work will involve iteration and refinement; however, there are also completely new features and partially designed concepts that will benefit from your creative input and ui/ux design skills.
+
+This role provides an opportunity to contribute to a 501c3 non-profit, open-source project that fosters meaningful connections around the world. You’ll work closely with frontend developers, design team members, and various other Couchers.org volunteers around the world.
+
+### Duties
+
+- Iterate on and refine existing Figma designs for both web and mobile experiences;
+- Design user flows, wireframes, and UI elements for new features as needed;
+- Conduct user testing and gather feedback to make data-informed design improvements;
+- Ensure all designs align with Couchers.org’s community guidelines for respect, safety, and inclusivity;
+- Collaborate with developers to implement design elements that are user-friendly and visually cohesive;
+- Adhere to and expand our design system based on Material UI standards.
+
+### Skills & Experience
+
+Ideal candidates will have the following:
+- A portfolio showcasing your experience in UI/UX design;
+- Proficiency in Figma and experience working within established design systems;
+- Familiarity with React and Material UI (our frontend stack) is a big plus;
+- Experience using or designing travel apps, ideally in the couch surfing or hospitality space;
+- Understanding of accessibility and usability best practices;
+- Strong communication skills in English (C1+) for effective cross-team collaboration;
+- Familiarity with github is a plus.
```

Comment:
Is the github necessary? Someone can learn that fast and might intimidate someone.